src/languageclient/reloadSchemas.ts

```
@@ -4,16 +4,16 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { ext } from '../extensionVariables';
-import { ReloadShemasRequest } from './ReloadSchemasRequest';
 
 export async function reloadSchemas(): Promise<void> {
 
-    ext.outputChannel.appendLine("Reloading schemas...");
-
     if (ext.languageServerClient) {
-        const request = new ReloadShemasRequest();
-        await ext.languageServerClient.sendRequest<ReloadShemasRequest>('schema/reloadSchemas', request);
+        ext.outputChannel.appendLine("Reloading schemas");
+
+        await ext.languageServerClient.sendRequest('schema/reloadSchemas', {});
+    }
+    else {
+        ext.outputChannel.appendLine("Language server is not yet ready");
```

You really should throw an Error because they won't see this message unless they have the window open.  The error will be caught and displayed automatically.  We can improve later.

---

Yep, had that change made already but didn't have it pushed. Pushed now.